user-service/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
```

Comment:
most of this could be part of globabl build.gradle

Reply:
i meant this one:

[Cross Configuration](https://docs.gradle.org/current/userguide/sharing_build_logic_between_subprojects.html#sec:convention_plugins_vs_cross_configuration
)

You may want to try  more "smart way"
[Convention Plugin](https://docs.gradle.org/current/userguide/sharing_build_logic_between_subprojects.html#sec:convention_plugins)
[sample](
https://docs.gradle.org/current/samples/sample_convention_plugins.html)

However i never did option #2, due to subprojects were well enough for my needs.